backend/src/main/java/com/bakdata/conquery/models/preproc/outputs/EpochDateRangeOutput.java

```diff
@@ -39,14 +40,13 @@ public Output createForHeaders(Object2IntArrayMap<String> headers) {
 		return new Output() {
 			@Override
 			protected Object parseLine(String[] row, Parser<?> type, long sourceLine) throws ParsingException {
-				if (!allowOpen && (row[startIndex] == null || row[endIndex] == null)) {
-					throw new IllegalArgumentException("Open Ranges are not allowed.");
-				}
-
-				if (row[startIndex] == null && row[endIndex] == null) {
+				if (Strings.isNullOrEmpty(row[startIndex]) && Strings.isNullOrEmpty(row[endIndex])) {
```

Comment:
Hier verstehe ich die Logik gerade nicht.

Müsstest du nicht erst prüfen ob beides  NullorEmpty ist und offene ranges erlaubt sind und `CDateRange.of()` zurückgeben anstatt `null`.

Die nachfolgende If-Bedingung, sollte die nicht als erstes geprüft werden?